testdata/dnn/download_models.py

```diff
@@ -1152,6 +1152,47 @@ def is_archive(self):
         ],
         sha='50008bb4f6a27b1aa940ad886b1bd1936ac4ed3e',
         filename='onnx/models/object_tracking_vittrack_2023sep.onnx'),
+    # Origian Intel Open Model Zoo: https://docs.openvino.ai/2023.3/omz_models_model_age_gender_recognition_retail_0013.html
+    Model(
+        name='age-gender-recognition-retail-0013-fp32 (xml)',
+        url='https://dl.opencv.org/models/intel_open_model_zoo/age-gender-recognition-retail-0013/FP32/age-gender-recognition-retail-0013.xml',
```

Comment:
Why not https://storage.openvinotoolkit.org/repositories/open_model_zoo/2023.0/models_bin/1/age-gender-recognition-retail-0013/FP32/ ?

Reply:
Added links to the original storage.